wallet/src/wallet/mod.rs

```diff
@@ -129,11 +131,13 @@ impl<B: storage::Backend> Wallet<B> {
 
         db_tx.commit()?;
 
+        let latest_median_time = chain_config.genesis_block().timestamp();
```

Comment:
Maybe I'm misunderstanding, but should this be getting the newest rather than the oldest timestamp?

Reply:
In this case I think it is all good, as it is a new wallet, only knows the genesis block, and it will get updated when it syncs from the node.

But I guess I need to update the one below when it is loading an existing wallet, in that case it can show a wrong balance in the short time between loading the wallet and the first sync with a node.